internal/backend/runtime/omni/controllers/omni/schematic_configuration.go

```diff
@@ -377,8 +385,8 @@ func detectExtensions(initialVersion, currentVersion semver.Version) ([]string,
 	if initialVersion.Major == 1 && initialVersion.Minor == 5 && currentVersion.GTE(semver.MustParse("1.6.0")) {
 		// install firmware
 		return []string{
-			"siderolabs/bnx2-bnx2x",
-			"siderolabs/intel-ice-firmware",
+			extensions.OfficialPrefix + "bnx2-bnx2x",
```

Comment:
i guess we can move this also to `extensions.MapByVersion`, follow somothing similar to quirks in talos, so all these customization is handled in one package